src/GraphQL.Client/GraphQLHttpClient.cs

```diff
@@ -143,7 +144,8 @@ private async Task<GraphQLHttpResponse<TResponse>> SendHttpRequestAsync<TRespons
             var preprocessedRequest = await Options.PreprocessRequest(request, this);
 
             using var httpRequestMessage = preprocessedRequest.ToHttpRequestMessage(Options, JsonSerializer);
-            using var httpResponseMessage = await HttpClient.SendAsync(httpRequestMessage, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
+            using var httpResponseMessage = await HttpClient.SendAsync(httpRequestMessage, HttpCompletionOption.ResponseHeadersRead, cancellationToken)
```

Comment:
Why only there and not two lines below also ?

Reply:
Deadlock happens exactly because of `HttpClient.SendAsync` has no `ConfigureAwait(false)`. To resolve issue #332 it's enougth to call `ConfigureAwait(false)` just here. But idealy whole project should be reviewed and each `await` should be configured. First, it will prevent from potential deadlocks, and second, it may have positive impact on productivity.

Reply:
> Deadlock happens exactly because of `HttpClient.SendAsync` has no `ConfigureAwait(false)`

Could you please write a test to verify this assertion ?

Reply:
Do you meen a unit test or just simple test application to be shure?

Reply:
Unit test to be able to run on CI.

Reply:
Yiu may call it unit or integration, whatever. The key part is to verify behavior on CI.

Reply:
I'm not sure there's somthing to test. Also I'm not sure it's posible to test `ConfigureAwait` or dedlock prevention. So I can't write it.